### DISCUSSION TITLE
Cannot follow Reverification Instructions

### DISCUSSION THREAD

**Discussion by stephendgilbert** [community/community#131971] [GitHub Education]
### Hi everyone,

I'm a teacher at a college in California. For about a month I've been receiving these messages (counting down to 1 day today) on my dashboard:
![image](https://github.com/user-attachments/assets/057374b8-6b80-4e4c-9228-6ddfa6d25de1)
When I click the link that says "reverify your academic affiliation", it takes me to this page.
![image](https://github.com/user-attachments/assets/5c76931a-fe70-40e1-9f98-f04e81edc7b7)
When I click the link in the corner, I don't see the Global Campus page. Instead, I'm taken to the main Education page where I'm asked to join GitHub Education (although I already have). When I search for Global Campus instead, I'm taken to a page that looks like this:
![image](https://github.com/user-attachments/assets/678fecdf-5d8f-4d2b-905a-2f6ca5f79503)
As you can see, there is nothing about "reverifying" my application.
What should I do?

**Comment by samus-aran** [comment]
Hi @stephendgilbert 👋 

The GitHub Education website has gone through some URL changes but you should be able to reverify yourself by going to the [application page](https://education.github.com/discount_requests/application), scroll down and begin your application (same as reverifying).

Let us know if you have any issues doing this.
